code/modules/reagents/chemistry/machinery/chem_heater.dm

```diff
@@ -478,6 +476,15 @@ To continue set your target temperature to 390K."}
 /obj/machinery/chem_heater/debug/Initialize()
 	. = ..()
 	reagents.maximum_volume = 2000
-	reagents.add_reagent(/datum/reagent/reaction_agent/basic_buffer, 980)
-	reagents.add_reagent(/datum/reagent/reaction_agent/acidic_buffer, 980)
+	reagents.add_reagent(/datum/reagent/reaction_agent/basic_buffer, 1000)
+	reagents.add_reagent(/datum/reagent/reaction_agent/acidic_buffer, 1000)
 	heater_coefficient = 0.4 //hack way to upgrade
+
+//map load types
```

Comment:
Use the `mapload` variable passed by Initialize instead IMO.

Reply:
You mean how it was before?

Reply:
No, the old one never used `mapload`--it's passed automatically to Initialize. Check if thats true, and add the reagents then.

Reply:
Sorry, I meant before coiax's request. I think you two are asking for conflicting things. Unless you mean add a mapload check to `/obj/machinery/chem_heater/withbuffer` ?

Reply:
Oh, my mistake, I didn't see that.

I'll defer to coiax's judgment.